docs/userguide/globalview/customizing-resource-interpreter.md

```diff
@@ -79,6 +79,7 @@ Supported resources:
 - StatefulSet(apps/v1)
 - Pod(v1)
 - PersistentVolumeClaim(v1)
+- PodDisruptionBudget(policy/v1)
```

Comment:
We also introduced `InterpretStatus` and `InterpretHealth` for PDB.

Reply:
done.